vpn/src/main/java/com/duckduckgo/mobile/android/vpn/blocklist/AppTrackerListUpdateWorker.kt

```diff
@@ -122,10 +121,9 @@ class AppTrackerListUpdateWorker(context: Context, workerParameters: WorkerParam
 @ContributesMultibinding(AppScope::class)
```

Comment:
We now need to explicitly tell the bound type, else it won't be picked up
```suggestion
@ContributesMultibinding(
    scope = AppScope::class,
    boundType = LifecycleObserver::class
)
```